### PR TITLE
Simplify Title area.

### PR DESCRIPTION
**Pull request by jsmith1024**

### CODE REVIEW

File: dox/books/Primer/doxyfile [hunk 1]
```diff
@@ -1064,7 +1064,7 @@ HTML_FILE_EXTENSION    = .html
 # of the possible markers and block names see the documentation.
 # This tag requires that the tag GENERATE_HTML is set to YES.
 
-HTML_HEADER            = ""
+HTML_HEADER            = "dox/new_header.html"
```
Comment (nekenyu) [inline dox/books/Primer/doxyfile]: Please don't add an unused feature to the dox/books/*/doxyfile

File: dox/new_header.html [hunk 2]
```diff
@@ -51,12 +51,5 @@
  </tr>
  </tbody>
 </table>
-</div>
```
Comment (nekenyu) [inline dox/new_header.html:54]: Please restore this &lt;/div&gt;.  If I compare the current version of this file against the default generated header html file, this line is missing.  It matches the &lt;div&gt; just before the &lt;table&gt; that was just closed.

**Review by nekenyu — CHANGES_REQUESTED** [review]

File: dox/new_header.html (hunk 2 above)
Comment (jsmith1024) [reply dox/new_header.html:54]: The double up of "Bill" looks clutterd and repetitive. If this solution is awkward, do you have an alternative suggeation?

File: dox/main.dox [hunk 3]
```diff
@@ -31,9 +31,9 @@ It is a general purpose statically typed, easy to learn and use language.
 </ul>
 
 <h4>License:</h4>
-    Copyright (c)  2022  J. Smith.</br>
-    Permission is granted to copy, distribute and/or modify this document, 
-    provided this notice is kept.</br>
+    <p>Copyright (c)  2022  Joe Smith.</p>
```
Comment (nekenyu) [inline dox/main.dox:34]: This is not consistent in formatting or text with dox/about.dox#license

**Review by nekenyu — CHANGES_REQUESTED** [review]

**Review by nekenyu — APPROVED** [review]

**Pull request merged**